metadata.yaml

```diff
@@ -63,7 +63,10 @@ provides:
     description: |
       Forwards the built-in grafana dashboard(s) for monitoring traefik.
     interface: grafana_dashboard
-
+  cert-transfer:
```

Comment:
Should we use terminology similar to `send-remote-write` and `receive-remote-write`?

Reply:
we can always rename that later, but what would you propose?
`send-certificate` feels OK, but we also have a `certificates` endpoint, should we call that `receive-certificate`? however that is not the counterpart to this one.
Do we want `generate-certificate`, `send-ceritficate`, `receive-certificate`? Feels a bit confusing. Let's agree on this another time :)